autogen/main/cluster.tf.tmpl

```diff
@@ -520,6 +520,14 @@ resource "google_container_node_pool" "pools" {
     }
 
     boot_disk_kms_key = lookup(each.value, "boot_disk_kms_key", "")
+
+    dynamic "kubelet_config" {
+      for_each = each.value.cpu_manager_policy[*]
```

Comment:
Not sure if this should be wrapped in a lookup because I don't know how maps work yet. Accessing a non-existent map key with . in the console seemed to be an error but the example I was iterating on seemed to pass the checks?

Reply:
Something like this should work
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/749#discussion_r533536750

Reply:
Thanks for bearing with me while I learn tf :D

Reply:
you are welcome. :slightly_smiling_face: 
But I know, using this dynamic block like this feels a little bit ugly at the beginning but you will see later why and how you want to avoid breaking changes. And this is a change where terraform will not inform you that it leads to a recreation of the whole node-pool instances.